deploy/examples/cluster-on-local-pvc.yaml

```diff
@@ -256,6 +256,10 @@ spec:
   #   requests:
   #      cpu: "200m"
   #      memory: "200Mi"
+  priorityClassNames:
+    mon: system-node-critical
+    osd: system-node-critical
```

Comment:
Should we call out here that mons/osds that can move nodes should be system-cluster-critical?

Reply:
I'll add a comment about that to the cluster-on-pvc.yaml where the OSDs are more likely to be portable. Here in the local PV cluster I wouldn't expected them to move.

Reply:
Oops. I meant to comment on `cluster-on-pvc`. 👍